third_party/terraform/tests/resource_container_node_pool_test.go.erb

```diff
@@ -785,7 +785,6 @@ resource "google_container_node_pool" "np" {
 `, cluster, np)
 }
 
-<% unless version == 'ga' -%>
```

Comment:
You'll also need to remove the version guards on the Test function that calls this one (line 35ish)

Reply:
Thanks!